content/modules/rest/pages/user-session-api.adoc

```diff
@@ -0,0 +1,141 @@
+= User Session API
+
+The User Session API provides information that is influenced by the current user session. It consists of the following two endpoints:
+
+* <<User Info>> Information about current user.
+* <<Permissions>> Information about entity permissions that the current user has obtained.
+
+== User Info
+
+The User Info Endpoint allows you to fetch information about the current user: `GET /userInfo` returns a `200 - OK` with the following information:
+
+[source,json]
+.Response: 200 - OK
+----
+{
+  "username": "admin",
+  "locale": "en"
+}
+----
+
+== Permissions
+
+The Permissions endpoint `/permissions` allows you to retrieve information about the permission that is applied to the current user session. In particular, the permissions contain information about the aspects:
+
+* Entity Permissions
+* Entity Attribute Permissions
+
+=== Entity Permissions
```

Comment:
I would remove this and the next headres, because the subsections are short enough and the example below is related to both of them.